src/components/sections/home/GetStarted.tsx

```diff
@@ -95,9 +97,32 @@ const GetStartedSteps: React.FC = () => {
 
 const GetStarted: React.FC = () => {
   const [yamlLink, setYamlLink] = useState(
-    "https://litmuschaos.github.io/litmus/litmus-operator-v1.11.2.yaml"
+    "https://litmuschaos.github.io/litmus/litmus-operator-v1.12.0.yaml"
   );
-  const initialValue = { yamlLink: yamlLink, setYamlLink: setYamlLink };
+  const [latestVersion, setLatestVersion] = useState<string>("");
+
+  useEffect(() => {
+    fetch("https://hub.litmuschaos.io/api/version")
+      .then((response) => response.json())
+      .then((data) => {
+        for (let i = 0; i < data.length; i++) {
+          // Iterating through all the releases to extract out the monthly ones
+          if (data[i].split(".")[2] === "0") {
+            setLatestVersion(data[i]); // Set the latest monthly release version
```

Comment:
data[0].slice(data[0].length - 2) + "0"

Can you try a smaller logic like this @S-ayanide

Reply:
Sure, this would shorten up the logic